proto/regen/data/v1/genesis.proto

```diff
@@ -15,8 +15,8 @@ message GenesisState {
 
 // GenesisContentEntry is a genesis content entry
 message GenesisContentEntry {
-  // hash is the ContentHash
-  ContentHash hash = 1;
+  // content_hash is the ContentHash
```

Comment:
I think we can delete this file. We are not using genesis types.

Reply:
https://github.com/regen-network/regen-ledger/pull/1014/commits/700eefcef6155ee55495e36515d1bd0d307be47d